rundeck/config.sls

```diff
@@ -90,13 +90,23 @@ rundeck.login:
 {% if 'sshkey' in rundeck_settings %}
 {% for dir,dir_options in rundeck_settings.sshkey.items() %}
 
+rundeck.sshkey.{{ dir }}:
+    file.directory:
+        - name: {{ dir }}
+        - user: {{ rundeck_settings.user }}
+        - group: {{ rundeck_settings.group }}
+        - mode: 700
```

Comment:
```suggestion
        - mode: '700'
```

I prefer the way you've done it but it's always best to stick to the convention of the surrounding code (i.e. the whole repo), so that means quoting here.  Consistency, essentially.

Reply:
Yes - I was caught a few times by `mode: 0700` translating to Hex so I always use the convention  `mode: '0700'`.

Reply:
@noelmcloughlin The quotes are mandatory [when using the leading zero](https://docs.saltstack.com/en/latest/ref/states/all/salt.states.file.html):


> ##### Warning
> 
> When using a mode that includes a leading zero you must wrap the value in single quotes. If the value is not wrapped in quotes it will be read by YAML as an integer and evaluated as an octal.

Reply:
Done. Sorry for lack of consistency, thanks for the review

Reply:
Actually, I wasn't expecting you to change all of these, since there are other places in the formula still using the 3-digit versions.  But since you've changed all of the places in this file, we can deal with the other places after this PR merge.

Reply:
Done in #9.